WordPress/Info.plist

```diff
@@ -33,6 +33,16 @@
 				<string>${WPCOM_SCHEME}</string>
 			</array>
 		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>com.optimizely</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>optly5635163020</string>
```

Comment:
I was prompted to add it by the Optimizely visual editor at one point early in the integration.  I've since disabled the visual editor for the iOS app, but neglected to clean up the URL scheme.  We can delete it.  I hadn't noticed what the number was until you pointed it out.  Props!